packages/server/graphql/mutations/addMissingJiraField.ts

```diff
@@ -87,12 +87,19 @@ const addMissingJiraField = {
     }
     const {fieldType, fieldId} = dimensionField
 
-    const screensResponse = await manager.getScreens(cloudId)
+    console.log(`Adding missing Jira field, fieldId: ${fieldId}, cloudId: ${cloudId}, projectKey: ${projectKey}`)
+
+    const maxResults = 2000
+    const screensResponse = await manager.getScreens(cloudId, maxResults)
     if (screensResponse instanceof Error || screensResponse instanceof RateLimitError) {
+      console.log('Unable to fetch screens for cloudId:', cloudId)
       return {error: {message: screensResponse.message}}
```

Comment:
-1 if we want to log this, I'd recommend something like `return standardError(screenResponse)`.

https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/server/utils/standardError.ts

Reply:
Fixed